src/dumpMetadata/controllers/dumpMetadataController.ts

```diff
@@ -57,4 +59,13 @@ export class DumpMetadataController {
 
     return res.status(httpStatus.OK).json(dumpsMetadata);
   };
+
+  public post: PostDumpMetadataHandler = async (req, res, next) => {
+    try {
+      await this.manager.createDumpMetadata(req.body);
```

Comment:
since we do not return the id of the dump metadata record, lets add a log message for indicating a successful creation with the id in it.

Reply:
done.